recipes/salt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "2015.8.7" %}
+
+package:
+  name: salt
+  version: {{ version }}
+
+source:
+  fn: salt-{{ version }}.tar.gz
+  url: https://github.com/saltstack/salt/archive/v{{ version }}.tar.gz
+  sha256: 93670d3a280e4a4d32fa12e2d5aaaa2ac7cb309c8f7fafff4dd8130fba7b40ff
+
+build:
+  number: 0
+  skip: True  # [win32 or py3k]
+
+requirements:
+  build:
```

Comment:
But they should be on the path so they shouldn't be required. They should be there via openssl. Since the `setup.py` is forcing this it feels like that is incorrect and a patch should be made upstream to allow this to be optional - since we satisfy that dependency ourselves. If you copy the files there then they will get shipped as part of the recipe and thus we are polluting the root of the conda environment with these two files.